spec/spec_helper.rb

```diff
@@ -1,9 +1,6 @@
 require "knapsack"
 Knapsack::Adapters::RSpecAdapter.bind
 
-require "webdrivers"
```

Comment:
I think it should probably be ok, but worth double checking everything works with `bundle exec rake parallel:spec` as expected

Reply:
So just tested running the parallel spec on both branches: on main it passes fine, on you're branch I had to explicitly install `chromedriver` and after that fully passes (which you have in docker compose) so all good